networkx/algorithms/cluster.py

```diff
@@ -27,13 +27,15 @@ def triangles(G, nodes=None):
     ----------
     G : graph
        A networkx graph
-    nodes : container of nodes, optional (default= all nodes in G)
+
+    nodes : container of nodes, optional (default = compute for all nodes in G)
```

Comment:
For me the potential inconsistency was that default value of `nodes` kwarg is `None`, but on a second read, the original phrasing is better.

Reply:
Yes -- I agree, it would be better to change the wording to be even more clear:
(adjust as you think is better)
```
    nodes : node, iterable of nodes, or None (default=None)
        If a singleton node, return the number of triangles for that node. 
        If an iterable, compute the number of triangles for each of those nodes.
        If `None` (the default) compute the number of triangles for all nodes in `G`.
```